tests/templates/test_subroutines_qmc.py

```diff
@@ -240,11 +240,11 @@ def test_expected_circuit(self):
         p = np.ones(4) / 4
         target_wires, estimation_wires = Wires(range(3)), Wires(range(3, 5))
 
-        with qml.tape.QuantumTape() as tape:
-            QuantumMonteCarlo(p, self.func, target_wires, estimation_wires)
+        op = QuantumMonteCarlo(p, self.func, target_wires, estimation_wires)
+        tape = op.expand().expand()
```

Comment:
This is needed because the operation expands to include `QuantumPhaseEstimation`, which itself can be expanded to more fundamental gates, as we expected in this original test.